src/code/z_skin.c

```diff
@@ -1,21 +1,222 @@
 #include "global.h"
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_skin/func_800A57C0.s")
+MtxF D_801600C0[60];
+char pad[0x10];
```

Comment:
`pad` is too generic for a global variable. I suggest leaving it as a D_ name for now, since that's what usually do for unknowns